.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
```

Comment:
Possibly contentious! We haven't discussed this with the wider team yet.

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length